File: 2022-2023/sourcecode/lab/lab7/GalleryAppJava/app/src/main/java/pl/udu/uwr/pum/galleryappjava/model/PictureModel.java
package pl.udu.uwr.pum.galleryappjava.model;

public class PictureModel {
    private int id = 0;
    private final String title;
    private final String image;

    public PictureModel(String title, String image) {
        this.title = title;
        this.image = image;
    }

    public PictureModel(int id, String title, String image){
        this(title, image);
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public String getImage() {
        return image;
    }
}

File: 2022-2023/sourcecode/lab/lab7/GalleryApp/app/src/main/java/pl/udu/uwr/pum/galleryapp/model/PictureModel.kt
package pl.udu.uwr.pum.galleryapp.model

data class PictureModel(val title: String, val image: String) {
    var id: Int = 0

    constructor(id: Int, title: String, image: String) : this(title, image) {
        this.id = id
    }
}